packit/upstream.py

```diff
@@ -68,6 +69,41 @@ def checkout_pr(self, pr_id: int) -> None:
         )
         self.local_project.git_repo.refs[f"pull/{pr_id}"].checkout()
 
+    def create_branch(self, branch_name: str, base: str = "HEAD", setup_tracking: bool = False) -> git.Head:
```

Comment:
It's probably time to create a common ancestor for distgit and upstream classes

Reply:
I guess this will be done in the other pull request, hopefully, right?
I don't want to mix two issues in one pull request.

Reply:
Please create an issue for it then.

Reply:
https://github.com/packit-service/packit/issues/157